packages/admin-panel/src/index.js

```diff
@@ -24,19 +28,26 @@ renderReactApp(
       {/* Store provider applied above routes so that it always persists auth state */}
       <StoreProvider api={api} persist>
         <EnvBanner />
-        <Switch>
-          <Route path="/viz-builder">
-            <VizBuilderProviders>
-              <VizBuilder Navbar={Navbar} Footer={Footer} />
-            </VizBuilderProviders>
-          </Route>
-          <Route path="/">
-            <AdminPanelProviders>
-              <AdminPanel />
-            </AdminPanelProviders>
-          </Route>
-          <Redirect to="/login" />
-        </Switch>
+        <StylesProvider injectFirst>
+          <MuiThemeProvider theme={theme}>
+            <ThemeProvider theme={theme}>
+              <CssBaseline />
```

Comment:
Pushed all the themes and css stuff up to this level, so that they wouldn't clash with the LESMIS ones when we import there.